source/2019-10-18-the-ember-times-issue-120.md

```diff
@@ -14,13 +14,15 @@ READMORE
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## Ember Bootstrap 3.0.0 released :tada:
```

Comment:
You can add a link to the header, so that a reader may click on the title to jump to a page that you want them to visit after reading. (We've had some trouble with CSS for header links though, so there may be readers who aren't aware of this feature.)
```suggestion
## [Ember Bootstrap 3.0.0 Released](https://www.ember-bootstrap.com) :tada:
```